analyzer/codechecker_analyzer/cmd/parse.py

```diff
@@ -375,7 +378,15 @@ def get_argparser_ctor_args():
         'epilog': """
 environment variables:
   CC_SEVERITY_MAP_FILE   Path of the checker-severity mapping config file.
+  CC_REPO_DIR         Root directory of the sources, i.e. the directory where
+                      the repository was cloned. Use it when generating gerrit
+                      output.
+  CC_REPORT_URL       URL where the report can be found. Use it when generating
+                      gerrit output.
+  CC_CHANGED_FILES    Path of changed files json from Gerrit. Use it when
+                      generating gerrit output.
```

Comment:
Where are these environment variables used in the code? I can't find it with grep under "analyzer" directory.

Reply:
They are used by the gerrit output converter here: codechecker_common/output/gerrit.py which is included by the analyzer part and the web part too.